docs/tutorial-create-pages.md

```diff
@@ -35,7 +35,7 @@ module.exports = HelloWorld;
 
 > Use any text editor to make the file, such as [Microsoft Visual Studio Code](https://code.visualstudio.com/download) or [Komodo Edit](https://www.activestate.com/komodo-edit).
 
-2. Go to http://localhost:3000/hello-world and you should be able to see the new page.
+2. Go to http://localhost:3000/docusaurus-tutorial/hello-world and you should be able to see the new page.
```

Comment:
This is only true after you have proceeded to add the `baseUrl` isn't it? When it's a fresh clone you don't see this.

Reply:
@yangshun You're right. I removed `docusaurus-tutorial/` from the links.